discatpy/models/permissions.py

```diff
@@ -8,6 +8,51 @@
 
 
 class Permissions(Flag):
+    if t.TYPE_CHECKING:
+        def __init__(
+            self,
+            create_instant_invite: bool = ...,
```

Comment:
And here

Reply:
[`b9449ce` (#18)](https://github.com/discatpy-dev/library/pull/18/commits/b9449ce65dede1ffe8bd714334dfb47f61533a39)